test/core/custom-page-sizes/custom-page-sizes-invalid.wast

```diff
@@ -108,3 +113,62 @@
   )
   "memory types incompatible"
 )
+
+;; Maximum memory sizes.
+;; These modules are valid, but instantiating them would allocate
+;; a huge memory, so test with `assert_unlinkable` + a missing import.
```

Comment:
Making a suggested rewording to this comment because it took me a little bit to understand the subtleties of this set up.

```suggestion
;; Maximum memory sizes.
;;
;; These modules are valid, but instantiating them is unnecessary
;; and would only allocate very large memories and slow down running
;; the spec tests. Therefore, add a missing import so that it cannot
;; be instantiated and use `assert_unlinkable`. This approach
;; enforces that the module itself is still valid, but that its
;; instantiation fails early (hopefully before any memories are
;; actually allocated).
```

Reply:
Also, since these modules are actually valid, it may make sense to remove these tests from this `custom-page-sizes-invalid.wast` file and move them to the `custom-page-sizes.wast` file (or alternatively to their own dedicated `max-sized-memory.wast` file).

Reply:
sgtm -- done.